### PR TITLE
Fix escape button when using search

### PR DESCRIPTION
**Pull request by danieljimeneznz**

### CODE REVIEW

**Comment by danieljimeneznz** [comment]
Partly fixes "Find in PDF" sections of #16

File: public/human-refinement/index.js [hunk 1]
```diff
@@ -380,6 +380,13 @@ const loadCase = async (caseId) => {
 		},
 	});
 
+	// Reset search params
+	$pdfSearchInput.value = '';
+	$pdfSearchInput.dataset.value = '';
+	$pdfSearchBar.classList.remove('active');
+	$pdfSearchInput.blur();
+	Array.from($('.searchDivPointWrap') || []).forEach((p) => p.remove());
```
Comment (danieljimeneznz) [inline public/human-refinement/index.js:388]: ^Would make the above a re-usable function but usage in scoping makes it annoying to use elsewhere... I can add it as window.resetSearch?

Comment (williamparry) [reply public/human-refinement/index.js:388]: That should be covered as part of https://github.com/openlawnz/openlawnz-human-refinement-centre/issues/16

**Comment by danieljimeneznz** [comment]
Close due to un-tested code and re-work

**Pull request closed without merging**